scripts/mysql_mongodb_converter/translation.rb

```diff
@@ -0,0 +1,35 @@
+table "districts" do
+	column "id", :key, :as => :integer, :rename_to => "code"
```

Comment:
I think it would be better if you use 2 spaces indentation for Ruby code as in https://github.com/bbatsov/ruby-style-guide#spaces-indentation.